sweet-spot-app/src/components/carousel-plugin.tsx

```diff
@@ -29,33 +29,38 @@ export function CarouselPlugin() {
       <CarouselContent>
         <CarouselItem>
           <div className="p-1">
-            <Card>
-              <CardContent className="flex h-48 items-center justify-center p-6 flex-col">
-                <UserPlus size={100} color="#008081" />
-                <p className="font-semibold">
-                  Create an account for your church to be able to add or remove
-                  members
+            <Card className="bg-[#f8f8f8]">
+              <CardContent className="flex h-36 items-center justify-end p-6 flex-col">
+                <MicVocal size={60} color="#000000" />
```

Comment:
would be cool to be able to use tailwind color here for readability

Reply:
good catch lol, I'll take care of that later!